src/common/snippets/src/pass/fq_decomposition.cpp

```diff
@@ -179,11 +174,13 @@ ngraph::snippets::pass::FakeQuantizeDecomposition::FakeQuantizeDecomposition() {
     register_matcher(m, callback);
 }
 
-bool ngraph::snippets::pass::FakeQuantizeDecomposition::isAllScalarConstant(const std::shared_ptr<const ngraph::Node>& node) {
-    return is_scalar_constant(node->get_input_node_shared_ptr(1)) &&
-           is_scalar_constant(node->get_input_node_shared_ptr(2)) &&
-           is_scalar_constant(node->get_input_node_shared_ptr(3)) &&
-           is_scalar_constant(node->get_input_node_shared_ptr(4));
+bool ngraph::snippets::pass::FakeQuantizeDecomposition::checkConstants(const std::shared_ptr<const ngraph::Node>& node) {
```

Comment:
Could you please clarify why have you rewritten the method?

Reply:
`FakeQuantizeDecomposition::isAllScalarConstant` is never used before. I removed the method and created new one, not rewrote.

Reply:
No we can remove this method due [PR#15837](https://github.com/openvinotoolkit/openvino/pull/15837) has been merged